mailet/base/src/main/java/org/apache/mailet/base/MailetUtil.java

```diff
@@ -21,19 +21,107 @@
 
 package org.apache.mailet.base;
 
+import static org.apache.mailet.base.MailetUtil.ValidationPolicy.ALL;
+
 import java.util.Optional;
+import java.util.function.Predicate;
+
 import javax.mail.MessagingException;
 
+import org.apache.james.util.OptionalUtils;
+import org.apache.james.util.Port;
 import org.apache.mailet.MailetConfig;
 
+import com.github.fge.lambdas.Throwing;
+import com.github.fge.lambdas.functions.FunctionChainer;
 import com.google.common.base.Strings;
 
 
 /**
  * Collects utility methods.
  */
 public class MailetUtil {
-    
+
+    public static class ValidationPolicy {
```

Comment:
either I don't understand or there's a naming issue : we are in MailetUtil.ValidationPolicy and nothing tells me we are talking about integers